app/controllers/eligibility_interface/countries_controller.rb

```diff
@@ -6,11 +6,20 @@ def new
       @country_form = CountryForm.new
     end
 
+    def next_url
```

Comment:
We should move these `next_url` methods in to the `private` section at the bottom of the class.